contracts/mainnet/connectors/morpho-rewards/main.sol

```diff
@@ -0,0 +1,100 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+import "./helpers.sol";
+import "./events.sol";
+
+/**
+ * @title Morpho Rewards.
+ * @dev Claim Morpho and Underlying Pool Rewards.
+ */
+
+abstract contract MorphoRewards is Helpers, Events {
+	/**
+	 * @dev Claim Pending MORPHO Rewards.
+	 * @notice Claims rewards.
+	 * @param _account The address of the claimer.
+	 * @param _claimable The overall claimable amount of token rewards.
+	 * @param _proof The merkle proof that validates this claim.
+	 */
+	function claimMorpho(address _account, uint256 _claimable, bytes32[] calldata _proof)
+		external
+		payable
+		returns (string memory _eventName, bytes memory _eventParam)
+	{
+		require(_proof.length > 0, "proofs-empty");
+
+		MORPHO_REWARDS.claim(_account, _claimable, _proof);
```

Comment:
Isn't the amount of claimed MORPHO tokens sotred somewhere in the storage via `setUint`?

Reply:
Agree I think it's missing too

Reply:
Yes! You're right. I've added it. My bad, I missed it.